mappings/net/minecraft/client/gl/GlShader.mapping

```diff
@@ -0,0 +1,33 @@
+CLASS none/bwn net/minecraft/client/gl/GlShader
+	CLASS none/bwn$a Type
+		FIELD a VERTEX Lnone/bwn$a;
+		FIELD b FRAGMENT Lnone/bwn$a;
+		FIELD c name Ljava/lang/String;
+		FIELD d fileExtension Ljava/lang/String;
+		FIELD e shaderType I
+		FIELD f nameObjectMap Ljava/util/Map;
+		METHOD <init> (Ljava/lang/String;ILjava/lang/String;Ljava/lang/String;I)V
+			ARG 0 name
+			ARG 1 fileExtension
+			ARG 2 shaderType
+		METHOD a getName ()Ljava/lang/String;
+		METHOD b getFileExtension ()Ljava/lang/String;
+		METHOD c getShaderType ()I
+		METHOD d getNameObjectMap ()Ljava/util/Map;
+	FIELD a shaderType Lnone/bwn$a;
+	FIELD b name Ljava/lang/String;
+	FIELD c shaderRef I
+	FIELD d refCount I
```

Comment:
Right, looking at the code and it being used it makes sense. Though `incRef` and `decRef` should maybe be renamed since they attach and delete the shader, too.